### PR TITLE
dev-ml/jbuilder: renamed to dev-ml/dune

### PR DESCRIPTION
**Pull request by AnAverageHuman**

### CODE REVIEW

**Comment by gyakovlev** [comment]
fun PR =)

btw, eclass changes we discussed on IRC merged. should work fine with EAPI7.

@aballier can you check this PR?

**Comment by Alessandro-Barbieri** [comment]
any progress on this?
@aballier ping

File: dev-ml/dune/dune-1.0_beta14.ebuild [hunk 1]
```diff
@@ -9,8 +9,8 @@ MY_PV="${PV/_/+}"
 MY_P="${PN}-${PV/_/-}"
 
 DESCRIPTION="A composable build system for OCaml"
-HOMEPAGE="https://github.com/janestreet/jbuilder"
-SRC_URI="https://github.com/janestreet/jbuilder/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+HOMEPAGE="https://dune.build"
+SRC_URI="https://github.com/ocaml/dune/archive/${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
```
Comment (mgorny) [inline dev-ml/dune/dune-1.0_beta14.ebuild]: I'd rather you reuse the old tarball and modify MY_P to continue working.

**Comment by Alessandro-Barbieri** [comment]
jbuilder was stable but dune isn't so it breaks stable ocaml packages

**Comment by AnAverageHuman** [comment]
> jbuilder was stable but dune isn't so it breaks stable ocaml packages

That's not the issue. dev-ml/dune was committed in b9d5d9070503414e7e4506d9097a8ba1355fd8f2 without the pkgmove. Then that was stabilized and now has stable deps.

**Pull request closed without merging**